paddle/operators/recv_op.cc

```diff
@@ -161,7 +161,6 @@ class RecvOpMaker : public framework::OpProtoAndCheckerMaker {
  public:
   RecvOpMaker(OpProto *proto, OpAttrChecker *op_checker)
       : OpProtoAndCheckerMaker(proto, op_checker) {
-    AddInput("RX", "(Tensor) Input tensor to be optimized").AsDuplicable();
```

Comment:
Delete the `Inputs` in RecvOp may cause all the book demo under `distributed_book` folder can not run.

Reply:
Done.